src/Sulu/Bundle/MediaBundle/Resources/js/containers/MultiMediaSelection/MultiMediaSelection.js

```diff
@@ -18,8 +19,6 @@ type Props = {|
     value: Value,
 |}
 
-const THUMBNAIL_SIZE = 'sulu-25x25';
```

Comment:
Since this value is used multiple times, it would make sense to keep this constant, wouldn't it?